doc/changes/unreleased.md

```diff
@@ -1,13 +1,35 @@
 # Unreleased
 
+## Summary
+
+From PyExasol version ``1.0.0``, the default behavior has been changed to use strict 
+certificate verification for encrypted connections (default, ``encryption=True``) in
```

Comment:
Encryption is probably already enabled by default, because the DB only accepts those connections. However, the default websock_sslopts have changed.